inspec.gemspec

```diff
@@ -11,48 +11,27 @@ Gem::Specification.new do |spec|
   spec.description   = "InSpec provides a framework for creating end-to-end infrastructure tests. You can use it for integration or even compliance testing. Create fully portable test profiles and use them in your workflow to ensure stability and security. Integrate InSpec in your change lifecycle for local testing, CI/CD, and deployment verification."
   spec.homepage      = "https://github.com/inspec/inspec"
   spec.license       = "Apache-2.0"
+  spec.require_paths = ["lib"]
 
-  # the gemfile and gemspec are necessary for appbundler so don't remove it
-  spec.files = %w{Gemfile inspec.gemspec README.md LICENSE} + Dir.glob(
-    "{bin,lib,etc}/**/*", File::FNM_DOTMATCH
-  ).reject { |f| File.directory?(f) }
+  spec.required_ruby_version = "~> 2.4"
 
-  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
-    .reject { |f| File.directory?(f) || f =~ %r{lib/plugins/.*/test/} }
-  spec.require_paths = ["lib"]
+  # ONLY the aws/azure/gcp files. The rest will come in from inspec-core
+  # the gemspec is necessary for appbundler so don't remove it
+  spec.files =
+    Dir.glob("{{lib,etc}/**/*,inspec.gemspec}")
+      .grep(/aws|azure|gcp|gemspec/)
+      .reject { |f| File.directory?(f) }
 
-  spec.required_ruby_version = ">= 2.4"
+  spec.add_dependency "inspec-core", "= #{Inspec::VERSION}"
```

Comment:
Doesn't this mean we always ship 2 gems?

Reply:
Any reason not to do something like this? https://github.com/chef/chef/blob/master/chef-universal-mingw32.gemspec#L1

Reply:
That's gross... I kinda like it.

Reply:
Also: 2 gems? We ship 3. 

https://rubygems.org/gems/inspec-bin
https://rubygems.org/gems/inspec-core
https://rubygems.org/gems/inspec

Reply:
We ship 4 gems. There is also inspec-core-bin.